config/prod/neo4j2owl-config.yaml

```diff
@@ -421,6 +421,7 @@ curie_map:
   EFO: http://www.ebi.ac.uk/efo/EFO_
   PATO: http://purl.obolibrary.org/obo/PATO_
   FBbt: http://purl.obolibrary.org/obo/FBbt_
+  FBdv: http://purl.obolibrary.org/obo/FBdv_
```

Comment:
@ubyndr, thank you for this. I'm going to review the CURIE map before I merge, because there may be others that were previously missed.